cmd/installer/root.go

```diff
@@ -1117,8 +1149,14 @@ func (p *PacksInstallationType) downloadPdscFile(pdscTag xml.PdscTag, skipInstal
 	}
 
 	utils.UnsetReadOnly(pdscFilePath)
+	os.Remove(pdscFilePath)
 	err = utils.MoveFile(localFileName, pdscFilePath)
 	utils.SetReadOnly(pdscFilePath)
+
+	if err != nil {
```

Comment:
It seems to me that this is superfluous :-)

Reply:
Remainder from previous error checking - removed.